java/test/jmri/jmris/json/JsonServerPreferencesPanelTest.java

```diff
@@ -11,23 +12,21 @@
  *
  * @author Paul Bender Copyright (C) 2017
  */
-public class JsonServerPreferencesPanelTest {
-
-    @Test
-    public void testCTor() {
-        JsonServerPreferencesPanel t = new JsonServerPreferencesPanel();
-        assertThat(t).withFailMessage("exists").isNotNull();
-    }
+public class JsonServerPreferencesPanelTest extends PreferencesPanelTestBase {
 
+    @Override
     @BeforeEach
     public void setUp() {
         JUnitUtil.setUp();
         jmri.util.JUnitUtil.resetProfileManager();
+        prefsPanel = new JsonServerPreferencesPanel();
     }
 
-    @AfterEach
-    public void tearDown() {
-        JUnitUtil.tearDown();
+    @Override
+    @Test
+    public void getLabelKey(){
+        // This class returns null for label key, but should it?
```

Comment:
The only preferences panels that should not be returning null label keys are preferences panels that are tabbed together with other preferences panels. Panels showing up in the list on the preferences window should be returning null for label keys.

Reply:
Thanks, will fix